analyzers/src/SonarAnalyzer.Common/SonarAnalyzer.Common.csproj

```diff
@@ -10,6 +10,10 @@
     <PackageReference Include="Google.Protobuf" Version="3.6.1" />
     <!-- When changing this reference update ProtocExe property too! -->
     <PackageReference Include="Google.Protobuf.Tools" Version="3.6.1" />
+    <PackageReference Include="Grpc.Tools" Version="2.26.0">
+      <PrivateAssets>all</PrivateAssets>
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+    </PackageReference>
```

Comment:
Needed to simplify the call to the protobuf compiler. Without it we have issues with the timing when the tool is called.